coordinator/src/main/scala/filodb.coordinator/queryplanner/SingleClusterPlanner.scala

```diff
@@ -395,8 +393,8 @@ class SingleClusterPlanner(dsRef: DatasetRef,
     val (renamedFilters, schemaOpt) = extractSchemaFilter(renameMetricFilter(lp.filters))
     val metaExec = shardsFromFilters(renamedFilters, options).map { shard =>
       val dispatcher = dispatcherForShard(shard)
-      SelectChunkInfosExec(options.queryId, options.submitTime, options.sampleLimit, dispatcher, dsRef, shard,
-        renamedFilters, toChunkScanMethod(lp.rangeSelector), schemaOpt, colName)
+      SelectChunkInfosExec(options, dispatcher, dsRef, shard, renamedFilters, toChunkScanMethod(lp.rangeSelector),
```

Comment:
Better to rename `options` as `qContext` in this file to avoid confusion for new reader